addons/l10n_ae/data/template/account.account-ae.csv

```diff
@@ -158,4 +158,5 @@
 "uae_account_500012","Excess In Till","500012","income","False"
 "uae_account_500013","Other Income","500013","income","False"
 "uae_account_500014","Cash Discount Gain","500014","income_other","False"
+"uae_account_999003","Off-Balance Sheet Counterpart Account","999003","off_balance","False"
```

Comment:
FYI this will be added only for new users, i don't know if a reloading of the chart template is really necessary since it's just one account ? @ygu-odoo Opinion on this ? Is the account really needed for everyone ? Specially that we do it in master so not sure